index.html

```diff
@@ -4,7 +4,403 @@
+            </section>
+          </li>
+          <li class="faq__item">
+            <input type="checkbox" id="faq__checkbox-4" name="faq__group" />
+            <label class="faq__heading" for="faq__checkbox-4"
+              >How do I access previous print magazines?
+            </label>
+            <section class="faq__content">
+              Contrary to popular belief, Lorem Ipsum is not simply random text.
+              It has roots in a piece of classical Latin literature from 45 BC,
+              making it over 2000 years old. Richard McClintock, a Latin
+              professor at Hampden-Sydney College in Virginia, looked up one of
+              the more obscure Latin words, consectetur, from a Lorem Ipsum
+              passage, and going through the cites of the word in classical
+              literature, discovered the undoubtable source.
+            </section>
+          </li>
+          <li class="faq__item">
+            <input type="checkbox" id="faq__checkbox-5" name="faq__group" />
+            <label class="faq__heading" for="faq__checkbox-5"
+              >What are the options for student memberships?
+            </label>
+            <section class="faq__content">
+              Contrary to popular belief, Lorem Ipsum is not simply random text.
+              It has roots in a piece of classical Latin literature from 45 BC,
+              making it over 2000 years old. Richard McClintock, a Latin
+              professor at Hampden-Sydney College in Virginia, looked up one of
+              the more obscure Latin words, consectetur, from a Lorem Ipsum
+              passage, and going through the cites of the word in classical
+              literature, discovered the undoubtable source.
+            </section>
+          </li>
+          <li class="faq__item">
+            <input type="checkbox" id="faq__checkbox-6" name="faq__group" />
+            <label class="faq__heading" for="faq__checkbox-6"
+              >HWhat is our cancellation and refund policy?
+            </label>
+            <section class="faq__content">
+              Contrary to popular belief, Lorem Ipsum is not simply random text.
+              It has roots in a piece of classical Latin literature from 45 BC,
+              making it over 2000 years old. Richard McClintock, a Latin
+              professor at Hampden-Sydney College in Virginia, looked up one of
+              the more obscure Latin words, consectetur, from a Lorem Ipsum
+              passage, and going through the cites of the word in classical
+              literature, discovered the undoubtable source.
+            </section>
+          </li>
+        </ul>
+      </section>
+    </main>
+    <footer class="footer">
+      <h2 class="footer__subtitle subtitle">
+        Join thousands of members gaining an edge with Digiday+
+      </h2>
+      <ul class="footer__list--top">
+        <li class="footer__item"></li>
```

Comment:
No no no no no, we don't forget about our unsighted friends. These should be pictures (`img` tag) with `alt` text.